libraries/standard/mqtt/cbmc/proofs/MQTT_Connect/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=MQTT_Connect_harness
+
+# Please see libraries/standard/mqtt/cbmc/stubs/network_interface_subs.c for
+# more information on MAX_NETWORK_SEND_TRIES.
+MAX_NETWORK_SEND_TRIES=3
+# Bound on the timeout in receiveConnack. This timeout is bounded because
+# memory saftey can be proven in a only a few iteration of the MQTT operations.
+# Each iteration will try to receive a single packet in its entirety. With a
+# time out of 3 we can get coverage of the entire function. Another iteration
+# performed will unnecessarily duplicate the proof.
+MQTT_RECEIVE_TIMEOUT=3
+DEFINES += -DMQTT_RECEIVE_TIMEOUT=$(MQTT_RECEIVE_TIMEOUT)
+DEFINES += -DMAX_NETWORK_SEND_TRIES=$(MAX_NETWORK_SEND_TRIES)
+INCLUDES +=
+
+# These functions do not coincide with the call graph of MQTT_Connect, but are
+# found by CBMC during processing in logs/MQTT_Connect_harness3.txt. We remove
+# the function bodies to increase the excution time of this proof.
```

Comment:
I suspect the real motivation is not execution time (execution misspelled), but coverage.  The existence of these functions that are provably unreachable lowers the coverage, but the solver can prove that they are unreachable blocks of code.

Reply:
I can reword this, I thought this might help.